elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/custom/DailyAverageScorePerPeriod.java

```diff
@@ -56,7 +56,7 @@ public QueryPlan resolve(Query query) {
                 .metricProjection(SQLMetricProjection.builder()
                         .alias(getAlias())
                         .name(getName())
-                        .expression(getExpression())
+                        .expression("AVG({{$highScore}})")
```

Comment:
Isn't this the same as before?  Why the change?

Reply:
added `$`